cpg_pipes/providers/cpg/status.py

```diff
@@ -104,12 +105,33 @@ def add_updaters_jobs(
             status=AnalysisStatus.COMPLETED,
             analysis_type=analysis_type,
             job_attrs=target.get_job_attrs(),
+            output=str(output),
```

Comment:
Type is `output: str | Path | Resource | dict[str, Path | Resource]`, but in the function it's asserted that it's a string, because the Analysis entries have the `output` field of type string as far as I know?

Reply:
I have it as a string, but a freeform, you can put anything you want in there (more undefined format than just str).

Should this be an exception or log an output is not of type Path - don't want it to accidentally drop outputs.

Reply:
Happy to populate anything into here! Just wasn't sure how difficult it would be to parse it in other applications if it was some random json